src/DSL/TSTryBlock.cs

```diff
@@ -12,7 +12,7 @@ public TSTryBlock(TSBuilder builder)
         {
         }
 
-        public TSTryBlock Catch(string errorName, Action<TSBlock> catchAction)
+        public void Catch(string errorName, Action<TSBlock> catchAction)
```

Comment:
no need for `finally`, basically?

Reply:
Yeah, until we need it, I'm going to pretend that doesn't exist.